glog 0.5.0: Compilation fails with Visual Studio and CMake

```
-- The C compiler identification is MSVC 19.29.30037.0
-- The CXX compiler identification is MSVC 19.29.30037.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30037/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30037/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR GTEST_MAIN_LIBRARY) 
-- Conan: Using autogenerated Findgflags.cmake
-- Found gflags: 2.2.2 (found suitable version "2.2.2", minimum required is "2.2.0") 
-- Library gflags_nothreads_static found C:/Users/runneradmin/.conan/data/gflags/2.2.2/_/_/package/7224b6b2a2459e7ad465c6dbbfff90f3a8f0a1a0/lib/gflags_nothreads_static.lib
-- Found: C:/Users/runneradmin/.conan/data/gflags/2.2.2/_/_/package/7224b6b2a2459e7ad465c6dbbfff90f3a8f0a1a0/lib/gflags_nothreads_static.lib
-- Looking for gflags namespace
-- Looking for gflags namespace - failed
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE  
-- Could NOT find Unwind (missing: Unwind_INCLUDE_DIR Unwind_LIBRARY Unwind_PLATFORM_LIBRARY) 
-- Looking for C++ include dlfcn.h
-- Looking for C++ include dlfcn.h - not found
-- Looking for C++ include execinfo.h
-- Looking for C++ include execinfo.h - not found
-- Looking for C++ include glob.h
-- Looking for C++ include glob.h - not found
-- Looking for C++ include inttypes.h
-- Looking for C++ include inttypes.h - found
-- Looking for C++ include memory.h
-- Looking for C++ include memory.h - found
-- Looking for C++ include pwd.h
-- Looking for C++ include pwd.h - not found
-- Looking for C++ include stdint.h
-- Looking for C++ include stdint.h - found
-- Looking for C++ include strings.h
-- Looking for C++ include strings.h - not found
-- Looking for C++ include sys/stat.h
-- Looking for C++ include sys/stat.h - found
-- Looking for C++ include sys/syscall.h
-- Looking for C++ include sys/syscall.h - not found
-- Looking for C++ include sys/time.h
-- Looking for C++ include sys/time.h - not found
-- Looking for C++ include sys/types.h
-- Looking for C++ include sys/types.h - found
-- Looking for C++ include sys/utsname.h
-- Looking for C++ include sys/utsname.h - not found
-- Looking for C++ include sys/wait.h
-- Looking for C++ include sys/wait.h - not found
-- Looking for C++ include syscall.h
-- Looking for C++ include syscall.h - not found
-- Looking for C++ include syslog.h
-- Looking for C++ include syslog.h - not found
-- Looking for C++ include ucontext.h
-- Looking for C++ include ucontext.h - not found
-- Looking for C++ include unistd.h
-- Looking for C++ include unistd.h - not found
-- Looking for C++ include ext/hash_map
-- Looking for C++ include ext/hash_map - not found
-- Looking for C++ include ext/hash_set
-- Looking for C++ include ext/hash_set - not found
-- Looking for C++ include ext/slist
-- Looking for C++ include ext/slist - not found
-- Looking for C++ include tr1/unordered_map
-- Looking for C++ include tr1/unordered_map - not found
-- Looking for C++ include tr1/unordered_set
-- Looking for C++ include tr1/unordered_set - not found
-- Looking for C++ include unordered_map
-- Looking for C++ include unordered_map - found
-- Looking for C++ include unordered_set
-- Looking for C++ include unordered_set - found
-- Looking for C++ include stddef.h
-- Looking for C++ include stddef.h - found
-- Check size of unsigned __int16
-- Check size of unsigned __int16 - done
-- Check size of u_int16_t
-- Check size of u_int16_t - failed
-- Check size of uint16_t
-- Check size of uint16_t - done
-- Looking for dladdr
-- Looking for dladdr - not found
-- Looking for fcntl
-- Looking for fcntl - not found
-- Looking for pread
-- Looking for pread - not found
-- Looking for pwrite
-- Looking for pwrite - not found
-- Looking for sigaction
-- Looking for sigaction - not found
-- Looking for sigaltstack
-- Looking for sigaltstack - not found
-- Performing Test HAVE_NO_DEPRECATED
-- Performing Test HAVE_NO_DEPRECATED - Failed
-- Performing Test HAVE_NO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_NO_UNNAMED_TYPE_TEMPLATE_ARGS - Failed
-- Looking for snprintf
-- Looking for snprintf - not found
-- Looking for UnDecorateSymbolName in dbghelp
-- Looking for UnDecorateSymbolName in dbghelp - found
-- Performing Test HAVE___ATTRIBUTE__
-- Performing Test HAVE___ATTRIBUTE__ - Failed
-- Performing Test HAVE___ATTRIBUTE__VISIBILITY_DEFAULT
-- Performing Test HAVE___ATTRIBUTE__VISIBILITY_DEFAULT - Failed
-- Performing Test HAVE___ATTRIBUTE__VISIBILITY_HIDDEN
-- Performing Test HAVE___ATTRIBUTE__VISIBILITY_HIDDEN - Failed
-- Performing Test HAVE___BUILTIN_EXPECT
-- Performing Test HAVE___BUILTIN_EXPECT - Failed
-- Performing Test HAVE___SYNC_VAL_COMPARE_AND_SWAP
-- Performing Test HAVE___SYNC_VAL_COMPARE_AND_SWAP - Failed
-- Performing Test HAVE_RWLOCK
-- Performing Test HAVE_RWLOCK - Failed
-- Performing Test HAVE___DECLSPEC
-- Performing Test HAVE___DECLSPEC - Success
-- Performing Test STL_NO_NAMESPACE
-- Performing Test STL_NO_NAMESPACE - Failed
-- Performing Test STL_STD_NAMESPACE
-- Performing Test STL_STD_NAMESPACE - Success
-- Performing Test HAVE_USING_OPERATOR
-- Performing Test HAVE_USING_OPERATOR - Success
-- Performing Test HAVE_NAMESPACES
-- Performing Test HAVE_NAMESPACES - Success
-- Performing Test HAVE_GCC_TLS
-- Performing Test HAVE_GCC_TLS - Failed
-- Performing Test HAVE_MSVC_TLS
-- Performing Test HAVE_MSVC_TLS - Success
-- Performing Test HAVE_CXX11_TLS
-- Performing Test HAVE_CXX11_TLS - Success
-- Performing Test HAVE_ALIGNED_STORAGE
-- Performing Test HAVE_ALIGNED_STORAGE - Success
-- Performing Test HAVE_CXX11_ATOMIC
-- Performing Test HAVE_CXX11_ATOMIC - Success
-- Performing Test HAVE_CXX11_NULLPTR_T
-- Performing Test HAVE_CXX11_NULLPTR_T - Success
-- Performing Test HAVE_SYMBOLIZE
-- Performing Test HAVE_SYMBOLIZE - Success
-- Performing Test HAVE_LOCALTIME_R
-- Performing Test HAVE_LOCALTIME_R - Failed
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Failed
-- Performing Test COMPILER_HAS_DEPRECATED
-- Performing Test COMPILER_HAS_DEPRECATED - Success
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/runneradmin/.conan/data/glog/0.5.0/bincrafters/testing/build/7fe4aa5e3a4e91c852b0aefcffd512ac654d1d2f
Conan:vcvars already set 
[1/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\demangle.cc.obj
[2/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\raw_logging.cc.obj
[3/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\vlog_is_on.cc.obj
[4/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\logging.cc.obj
[5/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\utilities.cc.obj
[6/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\symbolize.cc.obj
[7/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\windows\port.cc.obj
FAILED: source_subfolder/CMakeFiles/glog.dir/src/windows/port.cc.obj 
C:\PROGRA~2\MICROS~1\2019\ENTERP~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DGFLAGS_DLL_DECLARE_FLAG=GOOGLE_GLOG_DLL_DECL -DGFLAGS_DLL_DEFINE_FLAG=GOOGLE_GLOG_DLL_DECL -DGLOG_NO_ABBREVIATED_SEVERITIES -DGLOG_NO_SYMBOLIZE_DETECTION -Isource_subfolder\src -Isource_subfolder -Isource_subfolder\src\windows -IC:\Users\runneradmin\.conan\data\gflags\2.2.2\_\_\package\7224b6b2a2459e7ad465c6dbbfff90f3a8f0a1a0\include /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP2 /MT /O2 /Ob2 /DNDEBUG /showIncludes /Fosource_subfolder\CMakeFiles\glog.dir\src\windows\port.cc.obj /Fdsource_subfolder\CMakeFiles\glog.dir\glog.pdb /FS -c source_subfolder\src\windows\port.cc
source_subfolder\src\windows\port.cc(64): error C2084: function 'int snprintf(char *const ,const size_t,const char *const ,...)' already has a body
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt\stdio.h(1919): note: see previous definition of 'snprintf'
[8/9] Building CXX object source_subfolder\CMakeFiles\glog.dir\src\signalhandler.cc.obj
ninja: build stopped: subcommand failed.
```

As you can see, `snprintf` is not found in `cstdio` by CMake test (it's officially not part of C++98 standard, and glog is compiled with C++98 standard by default): https://github.com/google/glog/blob/c1499f64a8fb2eae2de79ed7195cf248e706efc9/CMakeLists.txt#L144

But when `port.cc` is compiled, definition of `snprintf` is found in `stdio.h`.

Reverting this test to include stdio.h instead of cstdio fixes this issue.

Here is CMakeError.log:

<details><summary>Click to expand log</summary>

```
Determining if the include file pthread.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_32bbd && [1/2] Building C object CMakeFiles\cmTC_32bbd.dir\CheckIncludeFile.c.obj

FAILED: CMakeFiles/cmTC_32bbd.dir/CheckIncludeFile.c.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo   /DWIN32 /D_WINDOWS /W3  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_32bbd.dir\CheckIncludeFile.c.obj /FdCMakeFiles\cmTC_32bbd.dir\ /FS -c CheckIncludeFile.c

CheckIncludeFile.c(1): fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file dlfcn.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_07817 && [1/2] Building CXX object CMakeFiles\cmTC_07817.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_07817.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_07817.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_07817.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'dlfcn.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file execinfo.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_d6e13 && [1/2] Building CXX object CMakeFiles\cmTC_d6e13.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_d6e13.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_d6e13.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_d6e13.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'execinfo.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file glob.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_6217a && [1/2] Building CXX object CMakeFiles\cmTC_6217a.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_6217a.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_6217a.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_6217a.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'glob.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file pwd.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_487f7 && [1/2] Building CXX object CMakeFiles\cmTC_487f7.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_487f7.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_487f7.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_487f7.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'pwd.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file strings.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_a7cfd && [1/2] Building CXX object CMakeFiles\cmTC_a7cfd.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_a7cfd.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_a7cfd.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_a7cfd.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'strings.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file sys/syscall.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_b5540 && [1/2] Building CXX object CMakeFiles\cmTC_b5540.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_b5540.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_b5540.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_b5540.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'sys/syscall.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file sys/time.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_857f0 && [1/2] Building CXX object CMakeFiles\cmTC_857f0.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_857f0.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_857f0.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_857f0.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'sys/time.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file sys/utsname.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_09aa8 && [1/2] Building CXX object CMakeFiles\cmTC_09aa8.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_09aa8.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_09aa8.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_09aa8.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'sys/utsname.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file sys/wait.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_75dfa && [1/2] Building CXX object CMakeFiles\cmTC_75dfa.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_75dfa.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_75dfa.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_75dfa.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'sys/wait.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file syscall.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_1a867 && [1/2] Building CXX object CMakeFiles\cmTC_1a867.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_1a867.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_1a867.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_1a867.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'syscall.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file syslog.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_9e97f && [1/2] Building CXX object CMakeFiles\cmTC_9e97f.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_9e97f.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_9e97f.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_9e97f.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'syslog.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file ucontext.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_0d8e5 && [1/2] Building CXX object CMakeFiles\cmTC_0d8e5.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_0d8e5.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_0d8e5.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_0d8e5.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'ucontext.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file unistd.h exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_7ff55 && [1/2] Building CXX object CMakeFiles\cmTC_7ff55.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_7ff55.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_7ff55.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_7ff55.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file ext/hash_map exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_a7f8a && [1/2] Building CXX object CMakeFiles\cmTC_a7f8a.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_a7f8a.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_a7f8a.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_a7f8a.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'ext/hash_map': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file ext/hash_set exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_7d753 && [1/2] Building CXX object CMakeFiles\cmTC_7d753.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_7d753.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_7d753.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_7d753.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'ext/hash_set': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file ext/slist exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_44c41 && [1/2] Building CXX object CMakeFiles\cmTC_44c41.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_44c41.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_44c41.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_44c41.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'ext/slist': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file tr1/unordered_map exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_70e9b && [1/2] Building CXX object CMakeFiles\cmTC_70e9b.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_70e9b.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_70e9b.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_70e9b.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'tr1/unordered_map': No such file or directory
ninja: build stopped: subcommand failed.




Determining if the include file tr1/unordered_set exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_1bcda && [1/2] Building CXX object CMakeFiles\cmTC_1bcda.dir\CheckIncludeFile.cxx.obj

FAILED: CMakeFiles/cmTC_1bcda.dir/CheckIncludeFile.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_1bcda.dir\CheckIncludeFile.cxx.obj /FdCMakeFiles\cmTC_1bcda.dir\ /FS -c CheckIncludeFile.cxx

CheckIncludeFile.cxx(1): fatal error C1083: Cannot open include file: 'tr1/unordered_set': No such file or directory
ninja: build stopped: subcommand failed.




Determining size of u_int16_t failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_86ac4 && [1/2] Building CXX object CMakeFiles\cmTC_86ac4.dir\HAVE_U_INT16_T.cpp.obj

FAILED: CMakeFiles/cmTC_86ac4.dir/HAVE_U_INT16_T.cpp.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_86ac4.dir\HAVE_U_INT16_T.cpp.obj /FdCMakeFiles\cmTC_86ac4.dir\ /FS -c C:\Users\spaceim\.conan\data\glog\0.5.0\_\_\build\7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4\CMakeFiles\CheckTypeSize\HAVE_U_INT16_T.cpp

C:\Users\spaceim\.conan\data\glog\0.5.0\_\_\build\7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4\CMakeFiles\CheckTypeSize\HAVE_U_INT16_T.cpp(29): error C2065: 'u_int16_t': undeclared identifier
C:\Users\spaceim\.conan\data\glog\0.5.0\_\_\build\7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4\CMakeFiles\CheckTypeSize\HAVE_U_INT16_T.cpp(30): error C2065: 'u_int16_t': undeclared identifier
C:\Users\spaceim\.conan\data\glog\0.5.0\_\_\build\7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4\CMakeFiles\CheckTypeSize\HAVE_U_INT16_T.cpp(31): error C2065: 'u_int16_t': undeclared identifier
C:\Users\spaceim\.conan\data\glog\0.5.0\_\_\build\7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4\CMakeFiles\CheckTypeSize\HAVE_U_INT16_T.cpp(32): error C2065: 'u_int16_t': undeclared identifier
C:\Users\spaceim\.conan\data\glog\0.5.0\_\_\build\7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4\CMakeFiles\CheckTypeSize\HAVE_U_INT16_T.cpp(33): error C2065: 'u_int16_t': undeclared identifier
ninja: build stopped: subcommand failed.



C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CheckTypeSize/HAVE_U_INT16_T.cpp:
#include <sys/types.h>
#include <stdint.h>
#include <stddef.h>


#undef KEY
#if defined(__i386)
# define KEY '_','_','i','3','8','6'
#elif defined(__x86_64)
# define KEY '_','_','x','8','6','_','6','4'
#elif defined(__PPC64__)
# define KEY '_','_','P','P','C','6','4','_','_'
#elif defined(__ppc64__)
# define KEY '_','_','p','p','c','6','4','_','_'
#elif defined(__PPC__)
# define KEY '_','_','P','P','C','_','_'
#elif defined(__ppc__)
# define KEY '_','_','p','p','c','_','_'
#elif defined(__aarch64__)
# define KEY '_','_','a','a','r','c','h','6','4','_','_'
#elif defined(__ARM_ARCH_7A__)
# define KEY '_','_','A','R','M','_','A','R','C','H','_','7','A','_','_'
#elif defined(__ARM_ARCH_7S__)
# define KEY '_','_','A','R','M','_','A','R','C','H','_','7','S','_','_'
#endif

#define SIZE (sizeof(u_int16_t))
static char info_size[] =  {'I', 'N', 'F', 'O', ':', 's','i','z','e','[',
  ('0' + ((SIZE / 10000)%10)),
  ('0' + ((SIZE / 1000)%10)),
  ('0' + ((SIZE / 100)%10)),
  ('0' + ((SIZE / 10)%10)),
  ('0' +  (SIZE    % 10)),
  ']',
#ifdef KEY
  ' ','k','e','y','[', KEY, ']',
#endif
  '\0'};

#ifdef __CLASSIC_C__
int main(argc, argv) int argc; char *argv[];
#else
int main(int argc, char *argv[])
#endif
{
  int require = 0;
  require += info_size[argc];
  (void)argv;
  return require;
}


Determining if the function dladdr exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_f694e && [1/2] Building C object CMakeFiles\cmTC_f694e.dir\CheckFunctionExists.c.obj

[2/2] Linking C executable cmTC_f694e.exe

FAILED: cmTC_f694e.exe 

cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\cmTC_f694e.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_f694e.dir\CheckFunctionExists.c.obj  /out:cmTC_f694e.exe /implib:cmTC_f694e.lib /pdb:cmTC_f694e.pdb /version:0.0 /machine:x64  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_f694e.dir\CheckFunctionExists.c.obj /out:cmTC_f694e.exe /implib:cmTC_f694e.lib /pdb:cmTC_f694e.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\cmTC_f694e.dir/intermediate.manifest CMakeFiles\cmTC_f694e.dir/manifest.res" failed (exit code 1120) with the following output:
CheckFunctionExists.c.obj : error LNK2019: unresolved external symbol dladdr referenced in function main

cmTC_f694e.exe : fatal error LNK1120: 1 unresolved externals

ninja: build stopped: subcommand failed.




Determining if the function fcntl exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_a1f34 && [1/2] Building C object CMakeFiles\cmTC_a1f34.dir\CheckFunctionExists.c.obj

[2/2] Linking C executable cmTC_a1f34.exe

FAILED: cmTC_a1f34.exe 

cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\cmTC_a1f34.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_a1f34.dir\CheckFunctionExists.c.obj  /out:cmTC_a1f34.exe /implib:cmTC_a1f34.lib /pdb:cmTC_a1f34.pdb /version:0.0 /machine:x64  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_a1f34.dir\CheckFunctionExists.c.obj /out:cmTC_a1f34.exe /implib:cmTC_a1f34.lib /pdb:cmTC_a1f34.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\cmTC_a1f34.dir/intermediate.manifest CMakeFiles\cmTC_a1f34.dir/manifest.res" failed (exit code 1120) with the following output:
CheckFunctionExists.c.obj : error LNK2019: unresolved external symbol fcntl referenced in function main

cmTC_a1f34.exe : fatal error LNK1120: 1 unresolved externals

ninja: build stopped: subcommand failed.




Determining if the function pread exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_4c5dc && [1/2] Building C object CMakeFiles\cmTC_4c5dc.dir\CheckFunctionExists.c.obj

[2/2] Linking C executable cmTC_4c5dc.exe

FAILED: cmTC_4c5dc.exe 

cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\cmTC_4c5dc.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_4c5dc.dir\CheckFunctionExists.c.obj  /out:cmTC_4c5dc.exe /implib:cmTC_4c5dc.lib /pdb:cmTC_4c5dc.pdb /version:0.0 /machine:x64  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_4c5dc.dir\CheckFunctionExists.c.obj /out:cmTC_4c5dc.exe /implib:cmTC_4c5dc.lib /pdb:cmTC_4c5dc.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\cmTC_4c5dc.dir/intermediate.manifest CMakeFiles\cmTC_4c5dc.dir/manifest.res" failed (exit code 1120) with the following output:
CheckFunctionExists.c.obj : error LNK2019: unresolved external symbol pread referenced in function main

cmTC_4c5dc.exe : fatal error LNK1120: 1 unresolved externals

ninja: build stopped: subcommand failed.




Determining if the function pwrite exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_9be68 && [1/2] Building C object CMakeFiles\cmTC_9be68.dir\CheckFunctionExists.c.obj

[2/2] Linking C executable cmTC_9be68.exe

FAILED: cmTC_9be68.exe 

cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\cmTC_9be68.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_9be68.dir\CheckFunctionExists.c.obj  /out:cmTC_9be68.exe /implib:cmTC_9be68.lib /pdb:cmTC_9be68.pdb /version:0.0 /machine:x64  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_9be68.dir\CheckFunctionExists.c.obj /out:cmTC_9be68.exe /implib:cmTC_9be68.lib /pdb:cmTC_9be68.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\cmTC_9be68.dir/intermediate.manifest CMakeFiles\cmTC_9be68.dir/manifest.res" failed (exit code 1120) with the following output:
CheckFunctionExists.c.obj : error LNK2019: unresolved external symbol pwrite referenced in function main

cmTC_9be68.exe : fatal error LNK1120: 1 unresolved externals

ninja: build stopped: subcommand failed.




Determining if the function sigaction exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_f2f33 && [1/2] Building C object CMakeFiles\cmTC_f2f33.dir\CheckFunctionExists.c.obj

[2/2] Linking C executable cmTC_f2f33.exe

FAILED: cmTC_f2f33.exe 

cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\cmTC_f2f33.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_f2f33.dir\CheckFunctionExists.c.obj  /out:cmTC_f2f33.exe /implib:cmTC_f2f33.lib /pdb:cmTC_f2f33.pdb /version:0.0 /machine:x64  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_f2f33.dir\CheckFunctionExists.c.obj /out:cmTC_f2f33.exe /implib:cmTC_f2f33.lib /pdb:cmTC_f2f33.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\cmTC_f2f33.dir/intermediate.manifest CMakeFiles\cmTC_f2f33.dir/manifest.res" failed (exit code 1120) with the following output:
CheckFunctionExists.c.obj : error LNK2019: unresolved external symbol sigaction referenced in function main

cmTC_f2f33.exe : fatal error LNK1120: 1 unresolved externals

ninja: build stopped: subcommand failed.




Determining if the function sigaltstack exists failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_69932 && [1/2] Building C object CMakeFiles\cmTC_69932.dir\CheckFunctionExists.c.obj

[2/2] Linking C executable cmTC_69932.exe

FAILED: cmTC_69932.exe 

cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\cmTC_69932.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100183~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_69932.dir\CheckFunctionExists.c.obj  /out:cmTC_69932.exe /implib:cmTC_69932.lib /pdb:cmTC_69932.pdb /version:0.0 /machine:x64  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_69932.dir\CheckFunctionExists.c.obj /out:cmTC_69932.exe /implib:cmTC_69932.lib /pdb:cmTC_69932.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\cmTC_69932.dir/intermediate.manifest CMakeFiles\cmTC_69932.dir/manifest.res" failed (exit code 1120) with the following output:
CheckFunctionExists.c.obj : error LNK2019: unresolved external symbol sigaltstack referenced in function main

cmTC_69932.exe : fatal error LNK1120: 1 unresolved externals

ninja: build stopped: subcommand failed.




Performing C++ SOURCE FILE Test HAVE_NO_DEPRECATED failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_5ae22 && [1/2] Building CXX object CMakeFiles\cmTC_5ae22.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_5ae22.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE_NO_DEPRECATED  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1   -Wdeprecated /showIncludes /FoCMakeFiles\cmTC_5ae22.dir\src.cxx.obj /FdCMakeFiles\cmTC_5ae22.dir\ /FS -c src.cxx

cl : Command line error D8021 : invalid numeric argument '/Wdeprecated'
ninja: build stopped: subcommand failed.



Source file was:
int main() { return 0; }
Performing C++ SOURCE FILE Test HAVE_NO_UNNAMED_TYPE_TEMPLATE_ARGS failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_e9608 && [1/2] Building CXX object CMakeFiles\cmTC_e9608.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_e9608.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE_NO_UNNAMED_TYPE_TEMPLATE_ARGS  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1   -Wunnamed-type-template-args /showIncludes /FoCMakeFiles\cmTC_e9608.dir\src.cxx.obj /FdCMakeFiles\cmTC_e9608.dir\ /FS -c src.cxx

cl : Command line error D8021 : invalid numeric argument '/Wunnamed-type-template-args'
ninja: build stopped: subcommand failed.



Source file was:
int main() { return 0; }
Determining if the snprintf exist failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_dae49 && [1/2] Building C object CMakeFiles\cmTC_dae49.dir\CheckSymbolExists.c.obj

FAILED: CMakeFiles/cmTC_dae49.dir/CheckSymbolExists.c.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo   /DWIN32 /D_WINDOWS /W3  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_dae49.dir\CheckSymbolExists.c.obj /FdCMakeFiles\cmTC_dae49.dir\ /FS -c CheckSymbolExists.c

C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\include\cstdio(33): error C2061: syntax error: identifier 'std'
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\include\cstdio(33): error C2059: syntax error: ';'
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\include\cstdio(33): error C2449: found '{' at file scope (missing function header?)
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\include\cstdio(91): error C2059: syntax error: '}'
CheckSymbolExists.c(14): fatal error C1004: unexpected end-of-file found
ninja: build stopped: subcommand failed.



File C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp/CheckSymbolExists.c:
/* */
#include <cstdio>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef snprintf
  return ((int*)(&snprintf))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Performing C++ SOURCE FILE Test HAVE___ATTRIBUTE__ failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_8fa95 && [1/2] Building CXX object CMakeFiles\cmTC_8fa95.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_8fa95.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE___ATTRIBUTE__  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_8fa95.dir\src.cxx.obj /FdCMakeFiles\cmTC_8fa95.dir\ /FS -c src.cxx

src.cxx(3): error C3646: '__attribute__': unknown override specifier
src.cxx(3): error C2059: syntax error: '('
src.cxx(3): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
ninja: build stopped: subcommand failed.



Source file was:

#include <cstdlib>
static void foo(void) __attribute__ ((unused));
int main(void) { return 0; }

Performing C++ SOURCE FILE Test HAVE___ATTRIBUTE__VISIBILITY_DEFAULT failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_4e569 && [1/2] Building CXX object CMakeFiles\cmTC_4e569.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_4e569.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE___ATTRIBUTE__VISIBILITY_DEFAULT  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_4e569.dir\src.cxx.obj /FdCMakeFiles\cmTC_4e569.dir\ /FS -c src.cxx

src.cxx(3): error C3646: '__attribute__': unknown override specifier
src.cxx(3): error C2059: syntax error: '('
src.cxx(3): error C2143: syntax error: missing ')' before '('
src.cxx(3): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
src.cxx(3): error C2440: 'initializing': cannot convert from 'const char [8]' to 'int'
src.cxx(3): note: There is no context in which this conversion is possible
src.cxx(3): error C2059: syntax error: ')'
ninja: build stopped: subcommand failed.



Source file was:

#include <cstdlib>
static void foo(void) __attribute__ ((visibility("default")));
int main(void) { return 0; }

Performing C++ SOURCE FILE Test HAVE___ATTRIBUTE__VISIBILITY_HIDDEN failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_8393a && [1/2] Building CXX object CMakeFiles\cmTC_8393a.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_8393a.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE___ATTRIBUTE__VISIBILITY_HIDDEN  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_8393a.dir\src.cxx.obj /FdCMakeFiles\cmTC_8393a.dir\ /FS -c src.cxx

src.cxx(3): error C3646: '__attribute__': unknown override specifier
src.cxx(3): error C2059: syntax error: '('
src.cxx(3): error C2143: syntax error: missing ')' before '('
src.cxx(3): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
src.cxx(3): error C2440: 'initializing': cannot convert from 'const char [7]' to 'int'
src.cxx(3): note: There is no context in which this conversion is possible
src.cxx(3): error C2059: syntax error: ')'
ninja: build stopped: subcommand failed.



Source file was:

#include <cstdlib>
static void foo(void) __attribute__ ((visibility("hidden")));
int main(void) { return 0; }

Performing C++ SOURCE FILE Test HAVE___BUILTIN_EXPECT failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_91809 && [1/2] Building CXX object CMakeFiles\cmTC_91809.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_91809.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE___BUILTIN_EXPECT  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_91809.dir\src.cxx.obj /FdCMakeFiles\cmTC_91809.dir\ /FS -c src.cxx

src.cxx(2): error C3861: '__builtin_expect': identifier not found
ninja: build stopped: subcommand failed.



Source file was:

int main(void) { if (__builtin_expect(0, 0)) return 1; return 0; }

Performing C++ SOURCE FILE Test HAVE___SYNC_VAL_COMPARE_AND_SWAP failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_0987d && [1/2] Building CXX object CMakeFiles\cmTC_0987d.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_0987d.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE___SYNC_VAL_COMPARE_AND_SWAP  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_0987d.dir\src.cxx.obj /FdCMakeFiles\cmTC_0987d.dir\ /FS -c src.cxx

src.cxx(4): error C3861: '__sync_val_compare_and_swap': identifier not found
ninja: build stopped: subcommand failed.



Source file was:

int main(void)
{
  int a; if (__sync_val_compare_and_swap(&a, 0, 1)) return 1; return 0;
}

Performing C++ SOURCE FILE Test HAVE_RWLOCK failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_9bde0 && [1/2] Building CXX object CMakeFiles\cmTC_9bde0.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_9bde0.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE_RWLOCK  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_9bde0.dir\src.cxx.obj /FdCMakeFiles\cmTC_9bde0.dir\ /FS -c src.cxx

src.cxx(3): fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory
ninja: build stopped: subcommand failed.



Source file was:

#define _XOPEN_SOURCE 500
#include <pthread.h>
int main(void)
{
  pthread_rwlock_t l;
  pthread_rwlock_init(&l, NULL);
  pthread_rwlock_rdlock(&l);
  return 0;
}

Performing C++ SOURCE FILE Test STL_NO_NAMESPACE failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_16687 && [1/2] Building CXX object CMakeFiles\cmTC_16687.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_16687.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DSTL_NO_NAMESPACE  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_16687.dir\src.cxx.obj /FdCMakeFiles\cmTC_16687.dir\ /FS -c src.cxx

src.cxx(3): error C2143: syntax error: missing ';' before '<'
src.cxx(3): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
ninja: build stopped: subcommand failed.



Source file was:

#include <vector>
vector<int> t; int main() { }

Performing C++ SOURCE FILE Test HAVE_GCC_TLS failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_280b0 && [1/2] Building CXX object CMakeFiles\cmTC_280b0.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_280b0.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE_GCC_TLS  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_280b0.dir\src.cxx.obj /FdCMakeFiles\cmTC_280b0.dir\ /FS -c src.cxx

src.cxx(2): error C2144: syntax error: 'int' should be preceded by ';'
src.cxx(2): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
ninja: build stopped: subcommand failed.



Source file was:

__thread int tls;
int main() { }

Performing C++ SOURCE FILE Test HAVE_LOCALTIME_R failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_e6bf5 && [1/2] Building CXX object CMakeFiles\cmTC_e6bf5.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_e6bf5.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DHAVE_LOCALTIME_R  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_e6bf5.dir\src.cxx.obj /FdCMakeFiles\cmTC_e6bf5.dir\ /FS -c src.cxx

src.cxx(8): error C3861: 'localtime_r': identifier not found
ninja: build stopped: subcommand failed.



Source file was:

#include <cstdlib>
#include <time.h>
int main()
{
    time_t timep;
    struct tm result;
    localtime_r(&timep, &result);
    return EXIT_SUCCESS;
}

Performing C++ SOURCE FILE Test COMPILER_HAS_DEPRECATED_ATTR failed with the following output:
Change Dir: C:/Users/spaceim/.conan/data/glog/0.5.0/_/_/build/7388bbc133673b4a6d4abbc3dc58a3c2c9fc3fa4/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~2/MICROS~2/2019/COMMUN~1/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_0a1ef && [1/2] Building CXX object CMakeFiles\cmTC_0a1ef.dir\src.cxx.obj

FAILED: CMakeFiles/cmTC_0a1ef.dir/src.cxx.obj 

C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.300\bin\Hostx64\x64\cl.exe  /nologo /TP -DCOMPILER_HAS_DEPRECATED_ATTR  /DWIN32 /D_WINDOWS /W3 /GR /EHsc  /MP8  /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_0a1ef.dir\src.cxx.obj /FdCMakeFiles\cmTC_0a1ef.dir\ /FS -c src.cxx

src.cxx(1): error C2065: '__deprecated__': undeclared identifier
src.cxx(1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
src.cxx(1): error C2062: type 'int' unexpected
src.cxx(1): error C2143: syntax error: missing ';' before '{'
src.cxx(1): error C2447: '{': missing function header (old-style formal list?)
src.cxx(2): error C3861: 'somefunc': identifier not found
ninja: build stopped: subcommand failed.



Source file was:
__attribute__((__deprecated__)) int somefunc() { return 0; }
    int main() { return somefunc();}
```

</details>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,15 +46,14 @@ set (CMAKE_CXX_VISIBILITY_PRESET hidden)
 set (CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-include (CheckCXXSourceCompiles)
 include (CheckCXXCompilerFlag)
 include (CheckCXXSourceCompiles)
 include (CheckCXXSourceRuns)
+include (CheckCXXSymbolExists)
 include (CheckFunctionExists)
 include (CheckIncludeFileCXX)
 include (CheckLibraryExists)
 include (CheckStructHasMember)
-include (CheckSymbolExists)
 include (CheckTypeSize)
 include (CMakePackageConfigHelpers)
 include (CMakePushCheckState)
@@ -141,7 +140,7 @@ check_cxx_compiler_flag (-Wunnamed-type-template-args
 
 # NOTE: Cannot use check_function_exists here since >=vc-14.0 can define
 # snprintf as an inline function
-check_symbol_exists (snprintf cstdio HAVE_SNPRINTF)
+check_cxx_symbol_exists (snprintf cstdio HAVE_SNPRINTF)
 
 check_library_exists (dbghelp UnDecorateSymbolName "" HAVE_DBGHELP)
 
@@ -446,7 +445,7 @@ add_definitions (-DGLOG_NO_SYMBOLIZE_DETECTION)
 
 check_cxx_source_compiles ("
 #include <cstdlib>
-#include <time.h>
+#include <ctime>
 int main()
 {
     time_t timep;
@@ -466,7 +465,7 @@ else (WITH_THREADS AND Threads_FOUND)
   set (NO_THREADS 1)
 endif (WITH_THREADS AND Threads_FOUND)
 
-# fopen/open on Cygwin can not handle unix-type paths like /home/.... 
+# fopen/open on Cygwin can not handle unix-type paths like /home/....
 # therefore we translate TEST_SRC_DIR to windows-path.
 if (CYGWIN)
   execute_process (COMMAND cygpath.exe -m ${CMAKE_CURRENT_SOURCE_DIR}

